practical-penguins/trivia_tavern/trivia_runner/models.py

```diff
@@ -0,0 +1,66 @@
+import random
+import string
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils import timezone
+
+from trivia_builder.models import TriviaQuiz, TriviaQuestion
+from phonenumber_field.modelfields import PhoneNumberField
+
+
+class Player(models.Model):
+    team_name = models.CharField(max_length=24, default='')
+    phone_number = models.CharField(max_length=12)
+    # Model name needs to be in quotes according to
+    # https://docs.djangoproject.com/en/3.0/ref/models/fields/#foreignkey
+    active_quiz = models.ForeignKey('ActiveTriviaQuiz', on_delete=models.CASCADE)
+
+    def get_answers(self):
+        answer_set = Answer.objects.filter(player=self)
+        answers = ""
+        for i, answer in enumerate(answer_set, start=1):
+            if answer.is_correct():
+                answers += f'Question {i}: your answer: {answer.value} is correct\n'
+            else:
+                answers += f'Question {i}: your answer: {answer.value} ' \
+                           f'does not match {answer.question.question_answer}\n'
+        return answers
+
+    def __str__(self):
```

Comment:
nice use of `__str__`

Reply:
I'm a huge fan of this `__str__`. It does a great job of actually describing the player object rather than simply printing an attribute. Descriptive `__str__`s and `__repr__`s can make a huge difference for debugging purposes.